docs/user-journeys/users/application-developer/intermediate.md

```diff
@@ -10,67 +10,136 @@ track: "USERS > APPLICATION DEVELOPER > INTERMEDIATE"
+There are pros and cons to each approach, though the *declarative approach* (e.g. `kubectl apply -f`) may be most helpful in production. Relying on local YAML files as the source of truth allows you to track and version control your configuration.
 
-Curabitur bibendum tempor mi, vel lacinia nisi vulputate ac. Nulla dignissim consectetur nisl nec tincidunt. Etiam pharetra facilisis sapien, non gravida velit fermentum sed. Ut ac ultrices nunc, in vestibulum urna. Suspendisse accumsan euismod felis, sit amet rhoncus neque volutpat luctus. Aliquam tincidunt pellentesque mauris, sed tempus diam. Mauris in elit eget justo tempor suscipit.
+For additional configuration best practices, familiarize yourself with [this guide](/docs/concepts/configuration/overview/){:target="_blank"}.
 
-Aenean suscipit arcu ac leo tincidunt tempus. Donec maximus tellus libero, ac ullamcorper magna lobortis ac. Integer mollis nisl vitae magna gravida, nec ornare ex consectetur. Sed mattis tincidunt nisi, at consequat tellus malesuada non. Integer vel semper nisi, ut fringilla velit. Nam felis ex, congue non dui vitae, sollicitudin convallis turpis. Phasellus porttitor maximus turpis, in varius nibh fermentum aliquam. Cras finibus lacus non diam porttitor porttitor. Nulla fringilla sagittis nibh nec condimentum. Duis egestas mauris nec dolor hendrerit ullamcorper sit amet in mi. Phasellus sollicitudin justo diam.
+To leverage pre-packaged configurations from the community, you may also be interested in checking out **{% glossary_tooltip text="Helm charts" term_id="helm-chart" %}**.
+
+#### Security
+
+You may be familiar with the principle of least privilege---if you are too generous with permissions when writing or using software, during a compromise, the negative effects can escalate out of control. Would you be cautious handing out `sudo` privileges to software on your OS? If so, you should be just as careful when granting your workload permissions to the Kubernetes API server (which is your cluster's source of truth!).
+
+You (or your {% glossary_tooltip text="cluster operator" term_id="cluster-operator" %}) can take care of this with the following:
+* **{% glossary_tooltip text="ServiceAccounts" term_id="service-account" %}** - An "identity" that your Pods can be tied to
+* **{% glossary_tooltip text="RBAC" term_id="rbac" %}** - One way of granting your ServiceAccount permissions
+
+[Authentication](/docs/admin/authentication/){:target="_blank"} and [Authorization](/docs/admin/authorization/){:target="_blank"} cover this topic more comprehensively, but are not mandatory reads.
+
+#### Resource isolation and management
+
+If your workloads may be operating in a **multi-tenant** environment with multiple teams or projects, your container(s) are *not* necessarily running alone on their node(s). They are sharing node resources with other containers which you do not own.
+
```

Comment:
You might also put in a link and mention Quality of Service, especially as Pod Priority is starting to get some serious traction in 1.9 and eviction/pre-emption could be a thing where the QoS rating makes a material difference in if your Pod's stay running vs. others with a better QoS assignment